cmd/helmvm/install.go

```diff
@@ -145,10 +145,11 @@ func createK0sctlConfigBackup(ctx context.Context) error {
 // updates the files that need to be uploaded to the nodes). This function also
 // makes sure that the k0s version used in the configuration matches the version
 // we are planning to install.
-func updateConfigBundle(ctx context.Context, bundledir string) error {
-	if err := createK0sctlConfigBackup(ctx); err != nil {
+func updateConfigBundle(c *cli.Context, bundledir string) error {
```

Comment:
Now that we are passing on the `cli.Context` we don't need to pass in `bundledir`, what do you think ? I believe that we also need to rephrase the comment for this function, it does not totally reflect what the function does. Maybe this function should be called `updateConfig` or something.